storm-client/src/jvm/org/apache/storm/daemon/supervisor/ClientSupervisorUtils.java

```diff
@@ -61,10 +61,15 @@ static boolean doRequiredTopoFilesExist(Map<String, Object> conf, String stormId
     }
 
     public static int processLauncherAndWait(Map<String, Object> conf, String user, List<String> args,
-                                             final Map<String, String> environment, final String logPreFix)
+                                             final Map<String, String> environment, final String logPreFix) throws IOException {
```

Comment:
does this "final" do anything here?

Reply:
I think it prevents the variable to be changed inside the method. Don't think it is very important so just left it as it is